internal/db/read_writer.go

```diff
@@ -533,6 +536,22 @@ func (rw *Db) SearchWhere(ctx context.Context, resources any, where string, args
 	return nil
 }
 
+// Now returns the current transaction timestamp.
```

Comment:
Mind adding a short explanation to the comment: `Now(...) will return the same time whenever it's called during the same transaction.  In other words, Now(...) at the start and Now(...) at the end of a transaction will always be equal.